src/components/page/Header/index.tsx

```diff
@@ -71,8 +71,8 @@ export class HeaderComponent extends React.Component<HeaderProps, State> {
     const { authenticated, user } = this.props;
     const path = this.props.location.pathname;
     return (
-      <div>
-        <Navbar light={true} expand="md">
+      <div className="row white-nav-bordered ">
+        <Navbar color="" light={true} expand="md">
```

Comment:
Can we drop the `color=""` prop since it's empty?

Reply:
Done